server/models/tenant.js

```diff
@@ -22,33 +22,33 @@ const tenantSchema = new mongoose.Schema(
     lease: {
```

Comment:
should we make the lease fields required? cause i thought a tenant is either renting, or DNE;

which in this case, if we are adding a tenant (that's just about to start living at this place) in the property home page, it should have these information available(filled out).

any thoughts? @nicholaslluong @hexxu 

we can leave the fees and payment history optional/empty.

Reply:
We will have to change the lease stuff back to required yes, as of right now since there is no way of adding lease information, we can technically never add a tenant, so i just changed it to optional for now.

Reply:
I was thinking of changing it back to required once you add the lease info to the tenant form :)

Reply:
i see. sounds good